pages/apim/3.x/event-native-apim/event-native-apim-user-guide.adoc

```diff
@@ -0,0 +1,243 @@
+[[event-native-apim-user-guide]]
+= User guide
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_user_guide.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+[NOTE]
+====
+This document assumes you are familiar with synchronous APIs, asynchronous APIs, and the OpenAPI specification.
+====
+
+[TIP]
+====
+Before running any of the examples on this page, work through some of the  link:event_native_apim_example_use_cases.html[example use cases] so you know that the event-native engine is running.
+====
+
+== Asynchronous APIs
+
+In this release, you can create asynchronous APIs and connect to event brokers and asynchronous data sources. You can configure asynchronous API connections using the RESTful interface to the management API.
+
+The OpenAPI specification for the management API is available at link:/apim/3.x/management-api/3.19/swagger.json[`swagger.json`] (link:/apim/3.x/apim_installguide_rest_apis_documentation.html#apim_console_api_reference[reference documentation]). Examples of how to use the management API to configure asynchronous APIs can be found in the link:https://www.postman.com/gravitee-io/workspace/gravitee-public-workspace/overview[Gravitee Postman Collections], which are described in link:event_native_apim_example_use_cases.html[example use cases].
```

Comment:
will be 3.20 link so, don't we have a `latest` ?

Reply:
Yes, once the collections are published, I will update the link to point directly to them.